src/service/get-all-videos.ts

```diff
@@ -5,12 +5,19 @@ import getAllVideosFromAPI from '../api-requests/get-all-videos';
 export const getAllVideos = async (): Promise<Video[]> => {
     const videos = await getAllVideosFromAPI(STRAPI_CLIENT);
     videos.forEach(v => {
-        v.otherTags = {
-            contentType: {
+        if (v.otherTags) {
```

Comment:
add content type to existing other tags for videos instead of creating fresh object